### DISCUSSION TITLE
Showing other Attributes as Entities (Sensors)

### DISCUSSION THREAD

**Discussion by ruffoxp-giu** [jeroenterheerdt/HAsmartirrigation#323] [Q&A]
Hi,

in readme section there is a paragraph with a link to explain how to show other attributes used for calculation, but the target page seems not exist [See the Wiki for more information on how to expose other values this integrations calculates as sensors](https://github.com/jeroenterheerdt/HAsmartirrigation/wiki/Showing-other-sensors)

Is it possible to have the correct link?

I need it to try to define a diagram that display historycal parameters, water needs and irrigation times.....something like this:

![image](https://github.com/jeroenterheerdt/HAsmartirrigation/assets/87600934/f6cef740-26a3-4914-b2f3-2b4aaac2cf85)

Thanku!

**Comment by jeroenterheerdt** [comment]
sorry, that info was outdated. You should be able to use templates to achieve what you want. Something like:
``` {{state_attr('sensor_name', 'attribute_name')}}```
